schema.json

```diff
@@ -1173,6 +1173,18 @@
          "force_reload": true,
          "id": "show_product_reviews_tabs"
       },
+      {
+         "type": "checkbox",
+         "label": "Product custom fields in tabs",
+         "force_reload": true,
+         "id": "show_custom_fields_tabs"
+      },
+      {
+        "type": "text",
+        "label": "Product custom fields tab label",
+        "force_reload": true,
```

Comment:
Since this setting isn't relevant if the above setting isn't checked, we should add

```
"reference": "show_custom_fields_tabs",
"reference_default": false,
```

This will cause this setting to be invisible if the above checkbox is not checked